Dockerfile

```
@@ -0,0 +1,30 @@
+FROM ubuntu:xenial
+MAINTAINER Alexander Paul <alex.paul@wustl.edu>
+
+LABEL \
+  version="1.0.5" \
```

Need to bump the version here:
```suggestion
  version="1.0.6" \
```